CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ This version changes some APIs in incompatible ways. For details how to migrate
  - C++ compiler requires C++20
  - In the C++ interpreter API `std::span` is used for callbacks arguments, instead of `sixtyfps::Slice`
  - `Model::row_data` will now return a `Option<T>` / `std::optional<T>` instead of a plain `T`.
+ - `Model::model_tracker` no longer has a default implementation.
+ - The depricated methods `Model::attach_peer` and `ModelNotify::attach` were removed.
```

Comment:
```suggestion
 - The deprecated methods `Model::attach_peer` and `ModelNotify::attach` were removed.
```

Reply:
@tronical already pointed that one out and I just committed his suggestion.